lzy/long-running/src/main/java/ai/lzy/longrunning/task/OperationTaskResolver.java

```diff
@@ -0,0 +1,36 @@
+package ai.lzy.longrunning.task;
+
+import ai.lzy.model.db.TransactionHandle;
+import jakarta.annotation.Nullable;
+
+import java.sql.SQLException;
+
+public interface OperationTaskResolver {
```

Comment:
Component that is used to match and create code representation to a task from DB.